app/src/test/java/comcmput301f16t01/github/carrier/AcceptingTests.java

```diff
@@ -0,0 +1,91 @@
+package comcmput301f16t01.github.carrier;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+/**
+ * Created by michael on 11/10/16.
+ */
+
+public class AcceptingTests {
+
+    /**
+     * (US 05.01.01) As a driver, I want to accept a request I agree with and accept that offered payment upon completion.
+     */
+    @Test
+    public void driverAccept(){
+        Rider rider = new Rider("Mike");
+        Request request = new Request(rider, new Location(), new Location());
+        RequestController rc = new RequestController();
+        rc.addRequest(request);
+        Driver driver = new Driver("Cole");
+        assertTrue("Get open request is not returning any open requests", rc.getOpenRequests().size() == 1);
+        rc.addDriver(request, driver);
+        assertTrue("The request is being updated when it shouldn't", rc.getOpenRequests().size() == 1);
+        assertTrue("The request has not been marked as accepted by the driver.", rc.getPendingRequests(driver).size() == 1);
```

Comment:
This is confusing to me. We are talking about requests and their status from two different perspectives: the rider and the driver. Based on the function name alone this is not clear to me which one we are talking about so the two tests were ambiguous. Also, "pending" is not a word that is well-defined within our application. This needs to be addressed in the Glossary.